CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 3.3.2 (2020-03-16)
+
+- Fixed the autocomplete component to properly highlight results after removing search text. [#65](https://github.com/blackbaud/skyux-lookup/pull/65)
```

Comment:
Can we combine the first two items? They seem pretty closely related. What do you think of `Fixed the autocomplete component to properly highlight results after removing search text and after only one character is supplied.`?

Reply:
And do we need "properly"?